From da6626debe00be1a0660f30cf2bf7629186c01d5 Mon Sep 17 00:00:00 2001
From: r-value <i@rvalue.moe>
Date: Tue, 16 Nov 2021 02:43:11 +0800
Subject: [PATCH] Remove incorrect i18n.merge_file argument

The positional argument was being silently ignored until meson 0.60.0 where
it fails with "ERROR: Function does not take positional arguments".
---
 data/meson.build | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index 2b5a0ee3..cf6e4ae1 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -8,16 +8,14 @@ install_data ('org.gnome.SimpleScan.gschema.xml',
               install_dir: join_paths (datadir, 'glib-2.0', 'schemas'))
 meson.add_install_script ('meson_compile_gschema.py')
 
-i18n.merge_file ('desktop-file',
-                 input: 'simple-scan.desktop.in',
+i18n.merge_file (input: 'simple-scan.desktop.in',
                  output: 'simple-scan.desktop',
                  install: true,
                  install_dir: join_paths (datadir, 'applications'),
                  po_dir: '../po',
                  type: 'desktop')
 
-i18n.merge_file ('appdata-file',
-                 input: 'simple-scan.appdata.xml.in',
+i18n.merge_file (input: 'simple-scan.appdata.xml.in',
                  output: 'simple-scan.appdata.xml',
                  install: true,
                  install_dir: join_paths (datadir, 'metainfo'),
-- 
GitLab

